Documentation/ApiOverview/CachingFramework/FrontendsBackends/Index.rst

```diff
@@ -658,8 +604,7 @@ Wincache Backend
 `Wincache <http://www.iis.net/downloads/microsoft/wincache-extension>`_ is a PHP opcode cache similar to APC, but
 dedicated to the Windows OS platform. Similar to APC, the cache can also be used as in-memory key/value cache.
 
-The cache backend implementation is nearly identical to the implementation of :ref:`APC backend <caching-backend-apc>`
-and has the same design constrains.
+The cache backend implementation is nearly identical to the implementation of `APC backend <https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/ApiOverview/CachingFramework/FrontendsBackends/Index.html#apc-backend>`_ and has the same design constrains.
```

Comment:
We should not use absolute urls, but sphinx references `:ref:` in order to keep links up to date even if we release a new version.